packages/lab/src/date-picker/DatePickerPanel.tsx

```diff
@@ -152,30 +154,32 @@ export const DatePickerPanel = forwardRef<HTMLDivElement, DatePickerPanelProps>(
             </FlexItem>
           )}
           <FlexLayout>
-            <Calendar
-              visibleMonth={startVisibleMonth}
-              onVisibleMonthChange={(_, month) => setStartVisibleMonth(month)}
-              {...firstCalendarProps}
-              {...CalendarProps}
-            />
-            {isRangePicker && (
+            <FormFieldContext.Provider value={{} as FormFieldContextValue}>
```

Comment:
Not too sure why this line is here. May be a subtle edge case needs to be fixed elsewhere?

Reply:
We need a comment, it's to avoid the Dropdown's in Calendar inheriting the FormField's state